Update Get-CsOnlineVoiceUser.md (#9205)
MicrosoftDocs/office-docs-powershell: skype/skype-ps/skype/Get-CsOnlineVoiceUser.md

```

## DESCRIPTION

> In Teams PowerShell Module version 3 and later, the following improvements have been introduced for organizations using Teams: 

Reviewer: ```suggestion
In Teams PowerShell Module version 3.0 and later, the following improvements have been introduced for organizations using Teams: 
```

> In the Teams PowerShell Module version 3 or later, the following improvements have been introduced for organizations having Teams: 

Reviewer 2: ```suggestion
In the Teams PowerShell Module version 3 or later, the following improvements have been introduced for organizations using Teams: 
```

> - This cmdlet now accurately returns users that are voice enabled (older cmdlet in 2.6.0 and below used to return users without MCOEV* plans assigned)

Reviewer 2: ```suggestion
- This cmdlet now accurately returns users who are voice-enabled (the older cmdlet in 2.6.0 and below returned users without MCOEV* plans assigned).
```

> - Result size is not limited to 100 users anymore (older cmdlet in 2.6.0 and below used to limit result size to 100)

Reviewer 2: ```suggestion
- The result size is not limited to 100 users anymore (the older cmdlet in version 2.6.0 and later limited the result size to 100).
```

> - This cmdlet now accurately returns users who are voice-enabled (the older cmdlet in version 2.6.0 and later returned users without MCOEV* plans assigned).
> - The result size is not limited to 100 users anymore (the older cmdlet in version 2.6.0 and later limited the result size to 100).

Reviewer 2: ```suggestion
- This cmdlet now accurately returns users who are voice-enabled (the older cmdlet in version 2.6.0 and earlier returned users without MCOEV* plans assigned).
- The result size is not limited to 100 users anymore (the older cmdlet in version 2.6.0 and earlier limited the result size to 100).
```

> In the Teams PowerShell Module version 2.6.2 or later, the following attributes have been deprecated for organizations having Teams users using the ExpandLocation parameter:

Reviewer 2: ```suggestion
In the Teams PowerShell Module version 2.6.2 and later, the following attributes have been deprecated for organizations with Teams users using the ExpandLocation parameter:
```

- ResultSize
- LicenceState

> In the Teams PowerShell Module version 2.6.2 or later, the following input parameters have been deprecated for organizations having Teams users due to low or zero usage:

Reviewer 2: ```suggestion
In Teams PowerShell Module version 2.6.2 and later, the following input parameters are deprecated for organizations with Teams users due to low or zero usage:
```
